solvers/deep_coral.py

```diff
@@ -37,30 +37,21 @@ def get_estimator(self, n_classes, device, dataset_name, **kwargs):
 
         dataset_name = dataset_name.split("[")[0].lower()
 
-        if dataset_name in ['mnist_usps']:
-            model = ShallowConvNet(n_classes=n_classes)
-        elif dataset_name in ['office31', 'officehome']:
-            # To change for a more suitable net
-            model = ShallowConvNet(n_classes=n_classes)
-        elif dataset_name in ['bci']:
-            # Use double precision for BCI data
-            model = ShallowMLP(input_dim=253, n_classes=n_classes).double()
-        else:
-            raise ValueError(f"Unsupported dataset: {dataset_name}")
-        
+        model, batch_size = get_model_batchsize_for_dataset(
+            dataset_name, n_classes)
+
         lr_scheduler = LRScheduler(
             policy='StepLR',
             step_every='epoch',
             step_size=1,
             gamma=0.7
         )
```

Comment:
where did you get this?

Reply:
It's in the torch example for MNIST, maybe not relevant for all datasets